CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change history for stripes-form
 
+## 8.0.0 IN PROGRESS
+
+* *BREAKING* bump `stripes-components`, `stripes-core` major versions.
```

Comment:
does this log belong to the changes in this PR? 
Also, the ref is not mentioned.

Reply:
It doesn't; I should have included the CHANGELOG entry in a previous PR. I will omit it here and restore it in a separate PR.

Huh? I don't think stripes-form uses any refs at all, and in any case this PR merely changes a dependency, not any code. What `ref` are you talking about?

Reply:
Do you mean the JIRA number `STFORM-28`?